LControl.py

```diff
@@ -1,7 +1,100 @@
 # This will be the main module that links up the GUI and https server
 
+import time
+import logging
+import sys, os
+from queue import Queue
+from threading import Thread
+# Add lib directory to module search path
+parent_dir = os.path.abspath(os.path.dirname(__file__))
+lib_dir = os.path.join(parent_dir, 'lib')
+
+# We insert our own libraries with our own modifications first
+# This way, the interpreter uses our libraries instead of the
+# ones that may exist on our machine
+sys.path.insert(0,lib_dir)
+import pyautogui
+
+loggingLevel = logging.INFO
+if getattr(sys, 'frozen', False):
+    # If the application is run as a bundle, the pyInstaller bootloader
+    # extends the sys module by a flag frozen=True and sets the app
+    # path into variable _MEIPASS'.
+    application_path = sys._MEIPASS
+    sys.release = True
+
+else:
+    loggingLevel = logging.DEBUG
+    application_path = os.path.dirname(os.path.abspath(__file__))
+    sys.release = False
+
+logging.getLogger()
+logging.basicConfig(format='%(levelname)s: %(message)s', level=loggingLevel)
+
+if sys.release:
+    logging.debug( "Failsafe disabled" )
+    pyautogui.FAILSAFE = False
+    pyautogui.PAUSE = 0
+
+elif not sys.release:
+    logging.info( "Failsafes enabled" )
+    logging.info( "Move mouse to upper-left corner to abort" )
+
+class LControl():
+    def __init__( self ):
+        self.q = Queue()
+        self.run = True
+
+    def moveDrag( self, x, y, move="move", duration=0.5 ):
+        # Line 821/822 are commented out in __init__.py for pyAutoGui
+        # This allows us to move between monitors
+        try:
+            x = int( x )
+            y = int( y )
+        except:
+            logging.warning( str( __name__ ) + " Received: " + str( x ) + " " + str( y ) )
+            return False
+
+        if move == "move":
+            logging.debug( "Moving to coordinates: " + str( x ) + " " + str( y ) )
+            pyautogui.moveRel( x, y, 0.5 )
+        elif move == "drag":
+            logging.debug( "Dragging to coordinates: " + str( x ) + " " + str( y ) )
+            pyautogui.dragRel( x, y, 0.5 )
+        else:
+            logging.error( str( __name__ ) + " Received: " + str( move ) )
+            return False
+
+        return True
+
+    def _serve_forever( self ):
```

Comment:
does python have a lambda or proc instead of having 2 similar method names calling eachother?

Reply:
Python has lambdas. The real purpose of starting a function with an underscore is python is to mark it as "private".  Private functions/variables do not exist in Python. It is just a convention. 

As for the 2 similar method names, I'm not quite sure if the public one will be renamed in the future with different usages.